test/sanity/ignore.txt

```diff
@@ -2003,6 +2003,7 @@ lib/ansible/modules/monitoring/zabbix/zabbix_template.py validate-modules:doc-re
 lib/ansible/modules/monitoring/zabbix/zabbix_template.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/monitoring/zabbix/zabbix_template_info.py validate-modules:doc-required-mismatch
 lib/ansible/modules/monitoring/zabbix/zabbix_user.py validate-modules:doc-required-mismatch
+lib/ansible/modules/monitoring/zabbix/zabbix_user_group.py validate-modules:doc-required-mismatch
```

Comment:
New ignores should not be added. Correct the issue with the module instead. If the issue is in a shared doc fragment, fix that in a separate PR first.

Reply:
@mattclay #65631

Reply:
Merged. Thanks.